src/main/scala/no/ndla/audioapi/model/api/PodcastMeta.scala

```diff
@@ -0,0 +1,25 @@
+package no.ndla.audioapi.model.api
+
+import org.scalatra.swagger.annotations.ApiModel
+import org.scalatra.swagger.runtime.annotations.ApiModelProperty
+
+import scala.annotation.meta.field
+
+// format: off
+@ApiModel(description = "Meta information about podcast audio")
+case class PodcastMeta(
+  @(ApiModelProperty @field)(description = "Header for the podcast") header: String,
+  @(ApiModelProperty @field)(description = "Introduction for the podcast") introduction: String,
+  @(ApiModelProperty @field)(description = "Cover photo for the podcast") coverPhoto: CoverPhoto,
+  @(ApiModelProperty @field)(description = "Manuscript for the podcast") manuscript: String,
```

Comment:
Dette vil gjerne være path til en fil og ikkje en faktisk tekst som lagres her. I tillegg har det vore diskutert mulighet for å ha både manuskript og transkribert tekst knytta til podcasten så då må det eventuellt legges til et felt for det.

Reply:
Når du sier path til en fil, er det en fil vi skal hoste noe sted? 
Trenger filopplasting til s3 e.l og url til filen som skal ligge her?

Reply:
Ja. Det er ikkje heilt spikra om det skal være url til s3 eller tekst som kan limes inn her, men String dekker jo begge behova.